src/Components/CreateImageWizard/ValidatedInput.tsx

```diff
@@ -89,6 +114,102 @@ export const HookPasswordValidatedInput = ({
   );
 };
 
+export const ValidatedInputAndTextArea = ({
+  value,
+  stepValidation,
+  fieldName,
+  onBlur,
+  placeholder,
+  onChange,
+  ariaLabel,
+  inputType = 'textInput',
+}: ValidationInputProp) => {
+  const errorMessage = stepValidation.errors[fieldName];
+  const hasError = errorMessage !== '';
+
+  return (
+    <>
+      <TextAndTextAreaInput
+        inputType={inputType}
+        ariaLabel={ariaLabel || ''}
+        value={value}
+        placeholder={placeholder || ''}
+        errorMessage={errorMessage}
+        onChange={onChange!}
+        onBlur={onBlur!}
+      />
+      {hasError && <ErrorMessage errorMessage={errorMessage} />}
+    </>
+  );
+};
+
+export const TextAndTextAreaInput = ({
```

Comment:
I'm looking at this component and wondering if we really need it. It's just a wrapper around the text input and the text area components with a blur handler and state management of `isPristine`. Could we just move `isPristine` up into ValidatedInputAndTextArea along with the blur handler and the useEffect and eliminate this component?

Reply:
yes, thanks